srcpkgs/instaloader/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'instaloader'
 pkgname=instaloader
-version=4.9.6
-revision=2
+version=4.10.1
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-requests"
 checkdepends="${depends}"
 short_desc="Download pictures with metadata from Instagram"
-maintainer="fosslinux <fosslinux@aussies.space>"
+maintainer="Jason Elswick <jason@jasondavid.tv>"
```

Comment:
we'd prefer if you had a few more contributions under your belt before adopting a package. Also, don't adopt packages that already have maintainers without asking them.